src/pages/api/modules/calendar.ts

```diff
@@ -45,6 +49,13 @@ async function Get(req: NextApiRequest, res: NextApiResponse) {
     (app) => app.integration && mediaAppIntegrationTypes.includes(app.integration.type)
   );
 
+  const IntegrationTypeEndpointMap = new Map<AppIntegrationType['type'], string>([
+    useSonarrv4 ? ['sonarr', '/api/v3/calendar'] : ['sonarr', '/api/calendar'],
```

Comment:
It would make more sense to write it the following way:
```suggestion
     ['sonarr', useSonarrv4 ? '/api/v3/calendar' : '/api/calendar'],
```